docs/installation-openshift-4.x/springboot/README.md

```diff
@@ -0,0 +1,113 @@
+#### Login into Openshift cluster
+```sh
+oc login -u system:admin
+eval $(minishift oc-env) 
```

Comment:
Again Minishift in Openshift 4.x readme?

Reply:
Fixed